dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/fields/FieldsPathParamsHelper.java

```diff
@@ -42,35 +41,30 @@
  * based on {@link FieldFilterParser }. It follows the principles of
  * {@link org.hisp.dhis.fieldfiltering.FieldFilterService}
  */
-public interface InstanceFieldsParamMapper
+public class FieldsPathParamsHelper
 {
-    String FIELD_RELATIONSHIPS = "relationships";
+    private FieldsPathParamsHelper()
+    {
+    }
 
-    String FIELD_EVENTS = "events";
+    protected static final String FIELD_RELATIONSHIPS = "relationships";
 
-    String FIELD_ATTRIBUTES = "attributes";
+    protected static final String FIELD_EVENTS = "events";
 
-    /**
-     * Parse the fields query parameter values to determine which resources
-     * should be fetched from the DB.
-     *
-     * @param fields fields query parameter values
-     * @return InstanceParams
-     */
-    InstanceParams map( List<String> fields );
+    protected static final String FIELD_ATTRIBUTES = "attributes";
 
-    static Map<String, FieldPath> getRoots( List<String> fields )
+    protected static Map<String, FieldPath> getRoots( List<String> fields )
```

Comment:
Do these need to be `protected`? If the class is package private these can only be accessed from within the package.